dbt/models/iasworld/schema/iasworld.land.yml

```diff
@@ -112,6 +112,16 @@ sources:
                   meta:
                     description: >
                       at least one major class (first digit) should match pardat class
+              - res_class_matches_pardat:
+                  name: iasworld_land_ex_matches_pardat_class_ex
+                  major_class_only: false
+                  join_type: inner
```

Comment:
Join counts with left are quite high. But, inner joins also make more sense conceptually.

Reply:
**[Praise]** Yup! We need the inner join here since otherwise the `class = 'EX'` filter on `pardat` will cause the majority of records in `land` to have no match in `pardat`, causing the test to compare the land class to `NULL`.